website/docs/how-to/unleash-edge-quickstart.md

```diff
@@ -0,0 +1,536 @@
+---
+title: Unleash Edge Quickstart Guide
+slug: /how-to/unleash-edge-quickstart
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+This document helps you get started with [Unleash Edge](https://docs.getunleash.io/reference/unleash-edge) locally.
+
+## Why Unleash Edge
+
+Unleash Edge as a lightweight layer between your SDKs and your Unleash instance.
+
+It exposes the same HTTP interface while providing higher throughput.
+
```

Comment:
I'm missing a bridging sentence here. Something like:
Before we dive into the overall setup, let’s look at the specific performance and resilience benefits you get from running Edge.